samples/react-native-macos/scripts/pod-install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+thisFilePath=$(dirname "$0")
+
+echo "USE_FRAMEWORKS=$USE_FRAMEWORKS"
+echo "ENABLE_PROD=$ENABLE_PROD"
+echo "ENABLE_NEW_ARCH=$ENABLE_NEW_ARCH"
+
+cd "${thisFilePath}/.."
+bundle install
+
+cd ios
```

Comment:
I think the `macos` [CI failure](https://github.com/getsentry/sentry-react-native/actions/runs/13392186006/job/37402281182?pr=4561) is related to this directory

```suggestion
cd macos
```

Reply:
Thanks.